test/zauction.test.ts

```diff
@@ -344,4 +334,43 @@ describe("zAuction Contract Tests", () => {
 
     await expect(tx).to.be.revertedWith("zAuction: data already consumed");
   });
+  it("Calculates minter royalty correctly", async () => {
+    // Royalty is set to 10% above
```

Comment:
Explicitly set the royalty in the mock in this test. Best practice is not to rely on previous tests for state.

Reply:
Specifically this refers to the setup in `before()` not a prior test, but I will change this